specification/botservice/resource-manager/Microsoft.BotService/preview/2017-12-01/botservice.json

```diff
@@ -796,6 +796,370 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.BotService/botServices/{resourceName}/listServiceProviders": {

```

Comment:
Why is this an action on the botService resource instead of an action on the provider itself? Are providers specific to an instance of the botService resource, likely not?

Reply:
made this an action on the botservice provider itself, since it's not specific to an instance of the resource